Prometheus.AspNetCore.HealthChecks/Prometheus.AspNetCore.HealthChecks.csproj

```diff
@@ -14,33 +15,25 @@
 		<Nullable>enable</Nullable>
 		<WarningsAsErrors />
 		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-	</PropertyGroup>
-
-	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-		<DocumentationFile>bin\Release\netstandard2.0\Prometheus.AspNetCore.HealthChecks.xml</DocumentationFile>
-		<CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
these properties were the same across all configurations, so I moved them to the top level for clarity.